x-pack/apm-server/profiling/collector.go

```diff
@@ -66,16 +66,15 @@ type ElasticCollector struct {
 
 	sourceFilesLock sync.Mutex
 	sourceFiles     *simplelru.LRU
+
+	clusterID string
 }
 
 // NewCollector returns a new ElasticCollector uses indexer for storing stack trace data in
 // Elasticsearch, and metricsIndexer for storing host agent metrics. Separate indexers are
 // used to allow for host agent metrics to be sent to a separate monitoring cluster.
-func NewCollector(
-	indexer elasticsearch.BulkIndexer,
-	metricsIndexer elasticsearch.BulkIndexer,
-	logger *logp.Logger,
-) *ElasticCollector {
+func NewCollector(indexer elasticsearch.BulkIndexer, metricsIndexer elasticsearch.BulkIndexer,
+	esClusterID string, logger *logp.Logger) *ElasticCollector {
```

Comment:
Ah the formatter messed this one up: do we have a style guideline for function arguments?

Reply:
No specific guidance/rules, but I rather liked it how it was ;)